modules/create-a-kubeletconfig-crd-to-edit-kubelet-parameters.adoc

```diff
@@ -60,12 +139,17 @@ metadata:
 spec:
   machineConfigPoolSelector:
     matchLabels:
-      custom-kubelet: large-pods
+      custom-kubelet: set-max-pods <1>
   kubeletConfig:
-    maxPods: 500
+    maxPods: 500 <2>
 ----
+<1> Enter the label from the Machine Config Pool.
+<2> Add the kubelet configuration. In this example, use `maxPods` to set the maximum pods per node.
 +
-The rate at which the kubelet talks to the API server depends on queries per second (QPS) and burst values. The default values, `50` for `kubeAPIQPS` and `100` for `kubeAPIBurst`, are good enough if there are limited pods running on each node. Updating the kubelet QPS and burst rates is recommended if there are enough CPU and memory resources on the node:
+[NOTE]
+====
+The rate at which the kubelet talks to the API server depends on queries per second (QPS) and burst values. The default values, `50` for `kubeAPIQPS` and `100` for `kubeAPIBurst`, are sufficient if there are limited pods running on each node. It is recommended to update the kubelet QPS and burst rates if there are enough CPU and memory resources on the node:
```

Comment:
This NOTE ends with a colon ("the node:") but there's nothing after it within the admonition box. Is the code block that follows it supposed to be up in the NOTE, or should that just be "the node."?